webrender/src/frame_builder.rs

```diff
@@ -786,9 +786,13 @@ impl FrameBuilder {
 
         let bs_rect = box_bounds.translate(box_offset)
                                 .inflate(inflate_amount, inflate_amount);
+        // If we have negative inflate amounts.
+        // Have to explicitly check this since euclid::TypedRect relies on negative rects
+        let bs_rect_empty = bs_rect.size.width <= 0.0 || bs_rect.size.height <= 0.0;
```

Comment:
No, but I put "negative" there since Euclid correctly handles 0 sized rects. I can update the comment to reflect that if you want. I was hoping to just do bs_rect.is_empty().